x-pack/plugins/snapshot_restore/__jest__/client_integration/home.test.ts

```diff
@@ -805,8 +812,12 @@ describe('<SnapshotRestoreHome />', () => {
 
               test('should display a message when snapshot in progress ', async () => {
                 const { find, actions } = testBed;
-                const updatedSnapshot = { ...snapshot1, state: 'IN_PROGRESS' };
-                httpRequestsMockHelpers.setGetSnapshotResponse(updatedSnapshot);
+                const updatedSnapshot = { ...snapshot2, state: 'IN_PROGRESS' };
```

Comment:
Was previously using the wrong snapshot mock but since the api call had a wildcard it was working fine.